forte/data/data_store.py

```diff
@@ -549,7 +555,15 @@ def get_attribute(self, tid: int, attr_name: str) -> Any:
         """
         try:
             entry = self.__entry_dict[tid]
-            entry_type = entry[constants.ENTRY_TYPE_INDEX]
+            if len(entry) > 3:
```

Comment:
Because only annotation-like entries have lengths larger than or equal to 4. if it is non-annotation-like, it has length 2. Should I use try-except to avoid magic numbers and avoid index out of bound error?